docs/releases/1.10.txt

```diff
@@ -214,6 +214,13 @@ Minor features
   ``{% load staticfiles %}`` or ``{% load static from staticfiles %}``) and
   not worry about whether or not the ``staticfiles`` app is installed.
 
+:mod:`django.utils.feedgenerator`
```

Comment:
I am not sure, I thought it is a modification. 

Should I put it in 1.11.txt and remove from 1.10.txt?